.github/workflows/action.yml

```diff
@@ -143,6 +143,9 @@ jobs:
       run: |
         composer config repositories.0 composer https://repo.magento.com
         composer config http-basic.repo.magento.com "$REPO_USR" "$REPO_PSW"
+        composer require --no-update nosto/module-nostotagging-cmp:@stable
```

Comment:
Should this be used `--install-suggests` instead?

Reply:
This same change needs to be done in the other repos as well.

Reply:
Good point, but I don't think `--install--suggests` fails if it can't be installed. Requiring forces to be installed and to fail if an error occurs.
There are also no docs about it: https://getcomposer.org/doc/03-cli.md#suggests